src/cmd_vel_mux.cpp

```diff
@@ -166,27 +221,120 @@ void CmdVelMux::configureFromParameters(const std::vector<std::string> & names,
   RCLCPP_INFO(get_logger(), "CmdVelMux : (re)configured");
 }
 
-void CmdVelMux::cmdVelCallback(const std::shared_ptr<geometry_msgs::msg::Twist> msg, unsigned int idx)
+bool CmdVelMux::addInputToParameterMap(std::map<std::string, ParameterValues> & parsed_parameters, const std::string & input_name, const std::string & input_variable, const rclcpp::Parameter & parameter_value)
```

Comment:
@clalancette @Lobotuerk consider `input_variable` -> `parameter_name`

Reply:
Fixed in 6091ab8532173b77448379318c2d7c1bb47dec2e